### PR TITLE
Bump minSdk to 21 across SDKs

### PR DESCRIPTION
**Pull request by emilypgoogle**

### CODE REVIEW

**Comment by rlazo** [comment]
Since this PR also changes the compileSdk, also include it in the description

File: appcheck/firebase-appcheck-safetynet/firebase-appcheck-safetynet.gradle [hunk 1]
```diff
@@ -26,7 +26,7 @@ android {
         timeOutInMs 60 * 1000
     }
 
-    compileSdkVersion project.targetSdkVersion
+    compileSdkVersion project.compileSdkVersion
```
Comment (rlazo) [inline appcheck/firebase-appcheck-safetynet/firebase-appcheck-safetynet.gradle:29]: This file no longer exists in the repo

File: appcheck/firebase-appcheck/ktx/ktx.gradle [hunk 2]
```diff
@@ -27,9 +27,9 @@ firebaseLibrary {
 
 android {
     namespace "com.google.firebase.appcheck.ktx"    
-    compileSdkVersion project.targetSdkVersion
+    compileSdkVersion project.compileSdkVersion
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion project.minSdkVersion
```
Comment (rlazo) [inline appcheck/firebase-appcheck/ktx/ktx.gradle:32]: You are also going to need to change the versions in the manifests, like for example https://github.com/firebase/firebase-android-sdk/blob/ep/min-sdk/appcheck/firebase-appcheck/src/main/AndroidManifest.xml

Comment (emilypgoogle) [reply appcheck/firebase-appcheck/ktx/ktx.gradle:32]: I decided to go through and change all the manifests, which was a lot of them, most of which were commented out and seemingly pinned to ancient versions

Comment (rlazo) [reply appcheck/firebase-appcheck/ktx/ktx.gradle:32]: Yes, thanks for updating them. They are meant to be commented out.

**Review by rlazo — APPROVED** [review]

**Pull request merged**